src/lib/viewers/controls/color-picker/ColorPickerToggle.scss

```diff
@@ -0,0 +1,59 @@
+@import '~box-ui-elements/es/styles/variables';
+@import '../styles';
+
+.bp-ColorPickerToggle {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 48px;
+    height: 48px;
+
+    .bp-ColorPickerToggle-button {
```

Comment:
Should these buttons be using the same mixins as the other existing buttons?

Reply:
Yep, they should.